src/test/java/seedu/address/testutil/TypicalStudent.java

```diff
@@ -46,17 +48,20 @@
             .withEmail("anna@example.com").build();
 
     // Manually added
-    public static final Student HOON = new StudentBuilder().withName("Hoon Meier").withTelegram("8482424")
+    public static final Student HOON = new StudentBuilder().withUuid("bd4f4c43-35d5-4afc-ab83-88cf20314560")
+            .withName("Hoon Meier").withTelegram("8482424")
             .withEmail("stefan@example.com").build();
-    public static final Student IDA = new StudentBuilder().withName("Ida Mueller").withTelegram("8482131")
+    public static final Student IDA = new StudentBuilder().withUuid("7f551343-cadb-4df3-8b50-76394fe84107")
+            .withName("Ida Mueller").withTelegram("8482131")
             .withEmail("hans@example.com").build();
 
     // Manually added - Student's details found in {@code CommandTestUtil}
```

Comment:
Perhpas this is missing a full-stop?
```suggestion
    // Manually added - Student's details found in {@code CommandTestUtil}.
```

Reply:
Resolved instead by changing to lowercase throughout, more in line with comments found above.